Source/JavaScriptCore/API/JSContext.mm

```diff
@@ -259,11 +259,11 @@ - (JSVirtualMachine *)virtualMachine
 
 - (NSString *)name
 {
-    auto name = adopt(JSGlobalContextCopyName(m_context));
+    SUPPRESS_UNCOUNTED_ARG auto name = adopt(JSGlobalContextCopyName(m_context));
```

Comment:
I don't understand this one. Since `name` is a smart pointer I wouldn't expect a diagnostic here. Maybe the issue is with the argument to the `adopt` function?